pyproject.toml

```
@@ -96,6 +96,7 @@ all = [
     "bottleneck>=1.3.3",
     "fsspec[http]>=2023.4.0",
     "s3fs>=2023.4.0",
+    "uncompresspy"
```

please set a lower bound now so we know where we're starting from
```suggestion
    "uncompresspy>=0.1.1"
```

---

Thanks, solved this in a later commit